build.gradle

```diff
@@ -21,5 +21,10 @@ allprojects {
     repositories {
         google()
     }
+
+    ext {
+        KEYSTORE_FILE = rootProject.file('scripts/key.jks')
+        CIRCLE_BUILD = System.getenv("CIRCLE") == "true" && KEYSTORE_FILE.exists()
```

Comment:
Show me where does it say that CircleCI Builds have `CIRCLE` as an environment variable

Reply:
Changed it to ```CIRCLECI```. This is present in the documentation.